.github/workflows/coding-standards.yml

```diff
@@ -0,0 +1,10 @@
+name: Coding Standards
+
+on:
+  pull_request:
+  push:
+
+jobs:
+  coding-standards:
+    name: Coding Standards
+    uses: brick/coding-standards/.github/workflows/coding-standards.yml@main
```

Comment:
Will this workflow run both ECS & Psalm from the coding standards repo, or do you want to separate them into different workflows? In the latter case, I'd prefer to keep the name `coding-style` if we're going to run ECS only.

I basically consider the coding standards repo as containing:
- `coding-style`: ECS
- `static-analysis`: Psalm

WDYT?

Reply:
Yeah agreed, let's add psalm workflow to this repository and properly split things like you mentioned.